Formula/redpanda.rb

```diff
@@ -14,6 +14,7 @@ class Redpanda < Formula
 
       def install
         bin.install "rpk"
+        generate_completions_from_executable(bin/"rpk", "generate", "shell-completion", base_name: "rpk")
```

Comment:
this file gets autogenerated by `goreleaser`. Apparently there is a way of getting the same effect ([see here](https://goreleaser.com/customization/homebrew/)) but needs to be done by tweaking the [goreleaser config file](https://github.com/redpanda-data/redpanda/blob/dev/src/go/.goreleaser.yaml)